python/ray/autoscaler/v2/instance_manager/common.py

```diff
@@ -249,6 +249,19 @@ def get_valid_transitions() -> Dict[
                 # cloud instance somehow failed.
                 Instance.TERMINATED,
             },
+            # Ray process should be stopped on the cloud instance. The RayStopper
+            # subscriber will listen to this status and stop the ray process.
+            Instance.RAY_STOP_REQUESTED: {
```

Comment:
Line 241 is also RAY_STOP_REQUESTED?

Reply:
We should also update other states that can transition into RAY_STOP_REQUESTED

Reply:
oh this change should actually be in https://github.com/ray-project/ray/pull/43078/files#diff-347f93d2e6bcd096619d2ec19be783eb03b5130999e395e5c4ebebc12d4c9cf4

I think there's some merge error where it gets double added here.